src/operators/pm_from_file.cc

```diff
@@ -25,6 +25,23 @@
 namespace modsecurity {
 namespace operators {
 
+bool PmFromFile::isComment(const std::string &s) {
+    if (s.size() == 0) {
+	  return true;
+    }
+    size_t pos = s.find("#");
+    if (pos != std::string::npos) {
+        for (int i = 0; i < pos; i++) {
+	    if (!std::isspace(s[i])) {
+		return false;
+	    }
+	}
+    } else {
+	return false;
+    }
+
+    return true;
+}
```

Comment:
Hi @tdoubley,

I think this specific commit is related to other issue :)  Should I take it in consideration? or is something that you are still working on?

Reply:
Hi @zimmerle  

I have just read the latest code ,  it‘s surly releated to [issue1948](https://github.com/SpiderLabs/ModSecurity/issues/1948), i think those issue can be solved by this commit.